### ISSUE TITLE
l-rdfToSparql: Javadoc warnings during compilation

### ISSUE THREAD

**Issue by jindrichmynarz** [UnifiedViews/Plugins#168]
When compiling l-rdfToSparql (develop branch, commit 88b2f651483ad759c56f613b6bf994ea5f576f05), Javadoc warnings are reported:

```
[WARNING] Javadoc Warnings
[WARNING] /Users/mynarzjindrich/Dev/UnifiedViews/Plugins/l-rdfToSparql/src/main/java/eu/unifiedviews/plugins/loader/rdftosparql/RdfToSparqlEndpointVaadinDialog.java:986: warning - Tag @link: reference not found: DPUConfigObject
[WARNING] /Users/mynarzjindrich/Dev/UnifiedViews/Plugins/l-rdfToSparql/src/main/java/eu/unifiedviews/plugins/loader/rdftosparql/RdfToSparqlEndpointVaadinDialog.java:1042: warning - Tag @link: reference not found: DPUConfigObject
[WARNING] /Users/mynarzjindrich/Dev/UnifiedViews/Plugins/l-rdfToSparql/src/main/java/eu/unifiedviews/plugins/loader/rdftosparql/RdfToSparqlEndpointVaadinDialog.java:986: warning - Tag @link: reference not found: DPUConfigObject
[WARNING] /Users/mynarzjindrich/Dev/UnifiedViews/Plugins/l-rdfToSparql/src/main/java/eu/unifiedviews/plugins/loader/rdftosparql/RdfToSparqlEndpointVaadinDialog.java:1042: warning - Tag @link: reference not found: DPUConfigObject
[WARNING] /Users/mynarzjindrich/Dev/UnifiedViews/Plugins/l-rdfToSparql/src/main/java/eu/unifiedviews/plugins/loader/rdftosparql/SPARQLoader.java:56: warning - Tag @link: reference not found: BaseRDFRepo
[WARNING] /Users/mynarzjindrich/Dev/UnifiedViews/Plugins/l-rdfToSparql/src/main/java/eu/unifiedviews/plugins/loader/rdftosparql/SPARQLoader.java:99: warning - @param argument "retrySize" is not a parameter name.
[WARNING] /Users/mynarzjindrich/Dev/UnifiedViews/Plugins/l-rdfToSparql/src/main/java/eu/unifiedviews/plugins/loader/rdftosparql/SPARQLoader.java:99: warning - @param argument "retryTime" is not a parameter name.
[WARNING] /Users/mynarzjindrich/Dev/UnifiedViews/Plugins/l-rdfToSparql/src/main/java/eu/unifiedviews/plugins/loader/rdftosparql/SPARQLoader.java:99: warning - @param argument "endpointParams" is not a parameter name.
[WARNING] /Users/mynarzjindrich/Dev/UnifiedViews/Plugins/l-rdfToSparql/src/main/java/eu/unifiedviews/plugins/loader/rdftosparql/SPARQLoader.java:99: warning - @param argument "username" is not a parameter name.
[WARNING] /Users/mynarzjindrich/Dev/UnifiedViews/Plugins/l-rdfToSparql/src/main/java/eu/unifiedviews/plugins/loader/rdftosparql/SPARQLoader.java:99: warning - @param argument "password" is not a parameter name.
[WARNING] /Users/mynarzjindrich/Dev/UnifiedViews/Plugins/l-rdfToSparql/src/main/java/eu/unifiedviews/plugins/loader/rdftosparql/SPARQLoader.java:129: warning - @param argument "endpointURL" is not a parameter name.
[WARNING] /Users/mynarzjindrich/Dev/UnifiedViews/Plugins/l-rdfToSparql/src/main/java/eu/unifiedviews/plugins/loader/rdftosparql/SPARQLoader.java:129: warning - @param argument "graphs" is not a parameter name.
[WARNING] /Users/mynarzjindrich/Dev/UnifiedViews/Plugins/l-rdfToSparql/src/main/java/eu/unifiedviews/plugins/loader/rdftosparql/SPARQLoader.java:129: warning - @param argument "userName" is not a parameter name.
[WARNING] /Users/mynarzjindrich/Dev/UnifiedViews/Plugins/l-rdfToSparql/src/main/java/eu/unifiedviews/plugins/loader/rdftosparql/SPARQLoader.java:129: warning - @param argument "password" is not a parameter name.
[WARNING] /Users/mynarzjindrich/Dev/UnifiedViews/Plugins/l-rdfToSparql/src/main/java/eu/unifiedviews/plugins/loader/rdftosparql/SPARQLoader.java:129: warning - @param argument "graphType" is not a parameter name.
[WARNING] /Users/mynarzjindrich/Dev/UnifiedViews/Plugins/l-rdfToSparql/src/main/java/eu/unifiedviews/plugins/loader/rdftosparql/SPARQLoader.java:129: warning - @param argument "insertType" is not a parameter name.
[WARNING] /Users/mynarzjindrich/Dev/UnifiedViews/Plugins/l-rdfToSparql/src/main/java/eu/unifiedviews/plugins/loader/rdftosparql/SPARQLoader.java:129: warning - @param argument "chunkSize" is not a parameter name.
[WARNING] /Users/mynarzjindrich/Dev/UnifiedViews/Plugins/l-rdfToSparql/src/main/java/eu/unifiedviews/plugins/loader/rdftosparql/WriteGraphType.java:21: warning - Tag @link: reference not found: GraphNotEmptyException
[WARNING] /Users/mynarzjindrich/Dev/UnifiedViews/Plugins/l-rdfToSparql/src/main/java/eu/unifiedviews/plugins/loader/rdftosparql/WriteGraphType.java:21: warning - Tag @link: reference not found: GraphNotEmptyException
[WARNING] /Users/mynarzjindrich/Dev/UnifiedViews/Plugins/l-rdfToSparql/src/main/java/eu/unifiedviews/plugins/loader/rdftosparql/RdfToSparqlEndpointVaadinDialog.java:986: warning - Tag @link: reference not found: DPUConfigObject
[WARNING] /Users/mynarzjindrich/Dev/UnifiedViews/Plugins/l-rdfToSparql/src/main/java/eu/unifiedviews/plugins/loader/rdftosparql/RdfToSparqlEndpointVaadinDialog.java:1042: warning - Tag @link: reference not found: DPUConfigObject
[WARNING] /Users/mynarzjindrich/Dev/UnifiedViews/Plugins/l-rdfToSparql/src/main/java/eu/unifiedviews/plugins/loader/rdftosparql/RdfToSparqlEndpointVaadinDialog.java:986: warning - Tag @link: reference not found: DPUConfigObject
[WARNING] /Users/mynarzjindrich/Dev/UnifiedViews/Plugins/l-rdfToSparql/src/main/java/eu/unifiedviews/plugins/loader/rdftosparql/RdfToSparqlEndpointVaadinDialog.java:1042: warning - Tag @link: reference not found: DPUConfigObject
```

**Comment by skrchnavy** [comment]
Part of #83

**Comment by tomas-knap** [comment]
This is really low prio issue, postponed

**Comment by tomas-knap** [comment]
Can be solved when #83 is done

**Comment by skrchnavy** [comment]
#83 is planed for 2.1.0 so it shall be synchronized

**Comment by skrchnavy** [comment]
postponed to next release

**Comment by mkl-eea-sk** [comment]
why is this here. it is "invalid" , "wontfix".

**Comment by skrchnavy** [comment]
@mkl-eea-sk please send positive message, where it shall be?

**Comment by mkl-eea-sk** [comment]
closed as wontfix. this issue will not exist after #83.

**Comment by skrchnavy** [comment]
OK when solved as a part of #83 then wioll be closed as resolved when #83 closed.
